.circleci/config.yml

```diff
@@ -930,6 +885,38 @@ workflows:
           requires:
             - assets_precompile
 
+  javascript_tests:
+    jobs:
+      - notify_start:
+          <<: *only-master-filter
+      - dependencies_bundler
+      - dependencies_npm
+      - assets_precompile:
+          requires:
+            - dependencies_bundler
```

Comment:
I'm not entirely sure, but I'd say we don't need `dependencies_bundler` for `karma` and `jest`. I'd at least run the jobs without them and confirm.

Reply:
I tried but `assets_precompile` requires `rake` and that's installed by bundler 🤦‍♂️

Reply:
You could use just ` dependencies_npm` job, no? or create a new one that won't require the bundler

Reply:
Like, not using `assets_precompile` for those (karma, jest). Lint will require it since `license_finder` requires it